tests/cross_functional/system_test/test_nfs_inout_export.py

```
@@ -0,0 +1,1188 @@
+        retcode, stdout, _ = con.exec_cmd(restored_checksum_cmd)
+        assert retcode == 0, f"Failed to calculate restored file checksum: {stdout}"
+
+        restored_file_checksum = stdout.split()[0]
+        log.info(f"Restored file checksum: {restored_file_checksum}")
+
+        # Get line count in restored file
+        restored_count_cmd = f"wc -l {restored_test_file}"
+        retcode, stdout, _ = con.exec_cmd(restored_count_cmd)
+        restored_line_count = int(stdout.split()[0]) if retcode == 0 else 0
+        log.info(f"Total lines in restored file: {restored_line_count}")
+        import pdb
```

Ruff (0.15.12)

[error] Import for `pdb` found

(T100)